### DISCUSSION TITLE
关于算法清单

### DISCUSSION THREAD

**Discussion by lingr7** [nuaatug/nuaathesis#75] [Ideas]
可能这个选项以前某个版本支持过？
但是现在的模板中，找不到可以设置的地方。
是否历史上有过支持，后续模板或者说规范修改了，又没有了。
![Snipaste_2021-11-21_16-19-08](https://user-images.githubusercontent.com/37898999/142754934-bd411c09-d4f6-4fa7-9d3f-5d5bb0382f3a.png)

**Comment by lingr7** [comment]
查看过一些记录，https://github.com/summershrimp/nuaathesis/blob/master/nuaathesis.cfg
这个文件中有提到
![Snipaste_2021-11-21_16-23-37](https://user-images.githubusercontent.com/37898999/142755016-43a3da4d-0d49-4cd0-abf5-2ef7791d2cb5.png)

**Reply by yzwduck** [reply]
你贴的代码来自 1.0 (实际 0.9a) 版本的模板，和目前维护的 2.x 版本有很大差异。

https://github.com/nuaatug/nuaathesis/blob/356c33bdef168d9aa535230fd363fb408e8747ca/nuaathesis.cfg#L16-L17

当时的模板有双语图/表标题的功能，结果有人反馈被导师抱怨了（抱歉，没找到 issue 号，但我记得有过这事），所以目前的策略是尽量与学校模板保持一致，不会输出代码清单。

**Reply by lingr7** [reply]
好的。我是出于功能性页数增加是好事的观点考量的。那没有就没有吧。
